bitbucket.md

```diff
@@ -0,0 +1,80 @@
+# Bitbucket
+
+1. Install Bitbucket
+ - https://www.atlassian.com/software/bitbucket/download
+
+1. Make a `Project` called `puppet` (with a short name of `PUP`)
+
+1. Create a repository called `control-repo`
+
+1. * Create a user called `r10k` with a password of `puppet`.
+   * Make the r10k user an admin of the `PUP` project.
+
+1. Either use the admin user to test pushing code, or create a user for yourself and add your SSH key to that user.
+  * If making a user for yourself, give your user account read/write or admin privilege to the `PUP` project.
+
+## Manually
+
+### Using The Post-Recieve Hook Plugin (Requires Java KS change)
```

Comment:
Used this method with a customer, it worked. Only missing step is that we needed to restart bitbucket to have it load the new bundle.